packages/near-membrane-base/src/connector.ts

```diff
@@ -26,7 +26,7 @@ const marshallSourceTextInStrictMode = `
                 // DEBUG_MODE.
                 // https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.debug_mode_enable
                 // eslint-disable-next-line @typescript-eslint/naming-convention
-                const DEBUG_MODE = function DEBUG_MODE() {}.name === 'DEBUG_MODE';
+                const DEBUG_MODE = `${() => /**/ 1}`.includes('*');
```

Comment:
why is string better than function?

Reply:
Via Sembian:
> if you have comments in the JS - the will be removed during minification. That could be a good check here I guess instead of function name.
>
>Minification - comments, whitespaces, unnecessary characters removed.
>
>Obfuscation - function, variable names modified typically to shorter names.

Reply:
> why is string better than function?

It's a function that is coerced to a string so we get the source which may, if-in-debug-mode contain the comment `/**/`, otherwise it's in prod mode.